jax/_src/lax/lax.py

```diff
@@ -4519,25 +4519,43 @@ def _dilate_shape(shape, dilation):
 def _ceil_divide(x1, x2):
   return -np.floor_divide(np.negative(x1), x2)
 
+
+jax_enum_names = [*[es.name for es in xla_client.PaddingType], 'PYTORCH_SAME']
+JaxPaddingType = enum.Enum(
```

Comment:
I think `PaddingType` would be a better name. It clearly is defined in JAX, as opposed to `xla_client.PaddingType`

Reply:
Renamed, thanks